storage/data-plane/src/main/java/com/microsoft/azure/storage/blob/PageBlobURL.java

```diff
@@ -324,7 +324,9 @@ public PageBlobURL withSnapshot(String snapshot) throws MalformedURLException, U
 
         String rangeString = pageRangeToString(range);
 
-        sourceOffset = sourceOffset ==  null ? 0 : sourceOffset;
+        if (sourceOffset == null) {
```

Comment:
Why did this change? I don't see this in the spotbugs report, and using the ternary operator is pretty standard in our sdk.

Reply:
The spotbugs complaints the box stuff. The ternary is trying to unboxing object to the primitive and box immediately. This is reported by plugins instead of command line. I will revert this back.

Reply:
Sorry, I ran the report again, it came back again. Could you run the spotbugs command line under central repo

Reply:
> At PageBlobURL.java:[line 327]
> Boxed value is unboxed and then immediately reboxed in com.microsoft.azure.storage.blob.PageBlobURL.uploadPagesFromURL(PageRange, URL, Long, byte[], PageBlobAccessConditions, SourceModifiedAccessConditions, Context)
> A boxed value is unboxed and then immediately reboxed.

Reply:
Not sure why the unboxing and reboxing is a problem, but I see it on the repot now. Thanks.

Reply:
To spotbugs point of view, it wastes time or memory to do like this.